src/injector.js

```diff
@@ -0,0 +1,103 @@
+const puma = require('pumascript');
+const http = require('http');
+const fs = require('fs');
+
+
+let errorsLogs = [];
+
+
+function readDependencyJsonFile(dir) {
+    try {
+        return JSON.parse(fs.readFileSync(dir, 'utf8'));
+    } catch (e) {
+        console.log('The file entered does not exist');
+    }
+    return undefined;
```

Comment:
you can define a variable let fileContent; at the top of the method, then you assign on the read file and return the variable always.

Reply:
If you just write return and do not specify the member it will be undefined. In other hand it's a bad practice return undefined dispite of the @emravera comment.

Reply:
Done.